From 2125d787ac14707510f71b501aeebcc159640541 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Thu, 12 Sep 2019 20:02:49 +0900
Subject: [PATCH] shared: remove unused spp_get_component_core

spp_get_component_core() is used for getting information of given lcore
ID, but not used from anywhere.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/shared/secondary/spp_worker_th/cmd_utils.c | 9 ---------
 src/shared/secondary/spp_worker_th/cmd_utils.h | 8 --------
 2 files changed, 17 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/cmd_utils.c b/src/shared/secondary/spp_worker_th/cmd_utils.c
index 2c50dcd..15ceb6f 100644
--- a/src/shared/secondary/spp_worker_th/cmd_utils.c
+++ b/src/shared/secondary/spp_worker_th/cmd_utils.c
@@ -499,15 +499,6 @@ spp_get_component_type(int id)
 	return component_info->wk_type;
 }
 
-/* Get core ID of target component */
-unsigned int
-spp_get_component_core(int component_id)
-{
-	struct sppwk_comp_info *info =
-			(g_mng_data.p_component_info + component_id);
-	return info->lcore_id;
-}
-
 /* Get core information which is in use */
 struct core_info *
 get_core_info(unsigned int lcore_id)
diff --git a/src/shared/secondary/spp_worker_th/cmd_utils.h b/src/shared/secondary/spp_worker_th/cmd_utils.h
index c778151..fb68905 100644
--- a/src/shared/secondary/spp_worker_th/cmd_utils.h
+++ b/src/shared/secondary/spp_worker_th/cmd_utils.h
@@ -204,14 +204,6 @@ int init_mng_data(void);
 /* Remove sock file if spp is not running */
 void del_vhost_sockfile(struct sppwk_port_info *vhost);
 
-/**
- * Get core ID of target component
- *
- * @param component_id Unique component ID.
- * @return Logical core id of specified component.
- */
-unsigned int spp_get_component_core(int component_id);
-
 /* Get core information which is in use */
 struct core_info *get_core_info(unsigned int lcore_id);
 
-- 
2.17.1

